functions.py

```diff
@@ -0,0 +1,162 @@
+import csv
+import datetime
+import exceptions_tests
+import os #this is needed for using directory paths and manipulating them
+import sys
+from pip._vendor.distlib.compat import raw_input
+
+class Employee(object):
+
+    def __init__(self, employee_id, name, phone, age):
+        self.employee_id = employee_id
+        self.name = name
+        self.phone = phone
+        self.age = age
+
+class Employees_list(object):
+    def __init__(self, list_of_employees):
+        self.list_of_employees = list_of_employees
+
+    def add_employee(self, new_employee):
+        """
+        This function gets the credentials of a new employee from user and adds it to the employees file.
```

Comment:
Pls add to this document the parameter that the function get.. (what new_employee is? and what type is it?)